findMinSubString/index.js

```diff
@@ -0,0 +1,41 @@
+ function minSubString(s, t) {
+     let n = s.length;
+     if (n == 0) {
+         return "np";
+     }
+     let mapt = {};
+     let maps = {};
+     t.split('').map(key => mapt[key] = mapt[key] || 0 + 1);
```

Comment:
Can you explain what this does? especially || 0+ 1 ?
mapt[key] = mapt[key] || 0 + 1);